node/README.md

```diff
@@ -8,14 +8,13 @@ Refer to the [Supported Engine Versions table](https://github.com/valkey-io/valk
 
 ## Current Status
 
-We've made Valkey GLIDE an open-source project, and are releasing it in Preview to the community to gather feedback, and actively collaborate on the project roadmap. We welcome questions and contributions from all Redis stakeholders.
-This preview release is recommended for testing purposes only.
+We've made Valkey GLIDE an open-source project, and are releasing it in Preview to the community to gather feedback, and actively collaborate on the project [roadmap](https://github.com/orgs/valkey-io/projects/11). We welcome questions and contributions from all Valkey stakeholders.
```

Comment:
we agreed to remove the current status: https://github.com/valkey-io/valkey-glide/pull/2299/files#r1771303639